lib/authorizer/digest.js

```diff
@@ -92,6 +102,23 @@ function _getDigestAuthHeader (headers) {
     });
 }
 
+/**
+ * Returns hex encoded hash of given data using given algorithm.
+ *
+ * @param {String} data string to calculate hash
+ * @param {String} algorithm hash algorithm
+ * @returns {String} hex encoded hash of given data
+ */
+function getHash (data, algorithm) {
+    // Current Electron version(7.2.3) in Postman app uses OpenSSL 1.1.0
+    // which don't support `SHA-512-256`. Use external `js-sha512` module
+    // to handle this case.
+    if (algorithm === 'sha512-256' && !_.includes(crypto.getHashes(), 'sha512-256')) {
```

Comment:
Move `!_.includes(crypto.getHashes(), 'sha512-256')` to top-level const and load `jsSHA = require('js-sha512')` in memory only if required.

Reply:
Done